Sources/SwifterSwift/SceneKit/SCNVector3Extensions.swift

```diff
@@ -29,6 +29,15 @@ public extension SCNVector3 {
     var length: SceneKitFloat {
         return sqrt(pow(x, 2) + pow(y, 2) + pow(z, 2))
     }
+    
+    /// SwifterSwift: Returns the unit or normalized vector where `length = 1`.
+    ///
+    ///     SCNVector3(2, 3, 6).normalized  -> SCNVector3(2/7, 3/7, 6/7)
+    ///
+    var normalized: SCNVector3 {
+        let len = sqrt(pow(x, 2) + pow(y, 2) + pow(z, 2))
+        return SCNVector3(x / len, y / len, z / len)
```

Comment:
I know it's nit-picky, but please use expressive variable names.
```suggestion
        let length = sqrt(pow(x, 2) + pow(y, 2) + pow(z, 2))
        return SCNVector3(x / length, y / length, z / length)
```